### DISCUSSION TITLE
6EB74087-3502-C384-D95C-66A96C929A9C

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#65] [General]
# Milky Way

The Milky Way is an SBc-type barred spiral galaxy with a isophotal diameter estimated at 87,400 light-years across. It is estimated to contain more than 100-400 billion stars, including the Sun itself.

This addon includes two models that depict its actual photographic view as seen from Earth and an artistic interpretation of what our galaxy might look like when viewed from the outside

https://celestia.mobi/addon?item=6EB74087-3502-C384-D95C-66A96C929A9C&lang=en

**Comment by starsfinderguy** [comment]
This is amazing

**Comment by kmjordan122313-lgtm** [comment]
my second favorite galaxy.
